odoo/addons/base/models/ir_mail_server.py

```diff
@@ -428,6 +428,7 @@ def _get_default_bounce_address(self):
         domain = get_param('mail.catchall.domain')
         if postmaster and domain:
             return '%s@%s' % (postmaster, domain)
+        return
```

Comment:
Can I ask why you added a "blank" return here?

Is it to have it explicit rather than implicit?

I guess we usually return "something" when pre-conditions are not fulfilled, such as "return False", not used to see it blank like than except when you want to break from the method execution (but in that case it's never at the end obviously).

Reply:
My linter yelled, that either we never return, either we should always return explicitly something. Actually it does not change anything, just making my linter happy here.